src/CategoriesContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  selectCategory,
+  loadRestaurants,
+} from './actions';
+
+import { get } from './utils';
+
+export default function CategoriesContainer() {
+  const dispatch = useDispatch();
+
+  const categories = useSelector(get('categories'));
+  const selectedCategory = useSelector(get('selectedCategory'));
+
+  function handleClick(categoryId) {
+    dispatch(selectCategory(categoryId));
+    dispatch(loadRestaurants());
+  }
+
+  return (
+    <ul>
+      {categories.map((category) => (
+        <li key={category.id}>
+          <button
+            type="button"
+            onClick={() => handleClick(category.id)}
+          >
+            {category.name}
+            {selectedCategory ? (
+              <>
+                {category.id === selectedCategory.id ? '(V)' : null}
+              </>
+            ) : null}
+          </button>
+        </li>
+      ))}
```

Comment:
category에서 id, name을 구조분해할당으로 가져와서 처리해줘도 좋을 것 같네요~
```suggestion
      {categories.map(({ id, name }) => (
        <li key={id}>
          <button
            type="button"
            onClick={() => handleClick(id)}
          >
            {name}
            {selectedCategory ? (
              <>
                {id === selectedCategory.id ? '(V)' : null}
              </>
            ) : null}
          </button>
        </li>
      ))}
```

- https://ko.javascript.info/destructuring-assignment